src/anomalib/models/padim/config.yaml

```diff
@@ -13,9 +13,11 @@ dataset:
   transform_config:
     train: null
     eval: null
-  test_split_mode: from_dir # options: [from_dir, synthetic]
+  test_split_mode: synthetic # options: [from_dir, synthetic]
+  test_synthetic_type: perlin_roi
```

Comment:
Maybe these should live under a separate key. Since `synthetic` is optional `test_synthetic_mode` probably shouldn't be at the same level as other keys.
How about?

```yaml
test_split:
    mode: synthetic/from_dir
    optional_key1: ...
    optional_key2: ...
    ...
```

Reply:
Hi Ashwin, thanks for the suggestion. We will work on the changes and keep you updated.

Reply:
hi @ashwinvaidya17 can we proceed with this design? We would like your approval before we make any changes as many codes will be impacted.

```
test_split:  
    mode: synthetic # options: [from_dir, synthetic]
      type: perlin_roi
    ratio: 0.2
```